examples/python/tests/drivers/test_options.py

```diff
@@ -29,3 +31,69 @@ def test_page_load_strategy_none():
 
     driver.get("http://www.google.com")
     driver.quit()
+
+def test_capabilities():
+    options = webdriver.ChromeOptions()
+    options.browser_version = 'stable'
+    options.platform_name = 'any'
+    options.accept_insecure_certs = True
+    driver = webdriver.Chrome(options=options)
+
+    driver.get("http://www.google.com")
```

Comment:
can we use 'https://www.selenium.dev/'

Reply:
I have updated them and will make sure to use selenium.dev moving forward.